Resources/Prototypes/Entities/Clothing/OuterClothing/suits.yml

```diff
@@ -8,6 +8,13 @@
     sprite: Clothing/OuterClothing/Suits/bombsuit.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Suits/bombsuit.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.15 #big blast protection, a little protection from heat, negligable cushioning from other damage cos its thick
```

Comment:
0.15 might be a liiiiittle too good, a specific 'explosion-resistance' thing might be better but out of scope obviously

Reply:
How about .2?

Reply:
sure